contrib/localnet/docker-compose.yml

```
@@ -241,6 +241,22 @@ services:
         ipv4_address: 172.20.0.103
     entrypoint: [ "/usr/bin/start-solana.sh" ]
 
+  ton:
+    image: ghcr.io/zeta-chain/ton-docker:latest
+    container_name: ton
+    hostname: ton
+    profiles:
+      - ton
+      - all
+    ports:
```

```suggestion
    platform: linux/amd64
    ports:
```
This fix an error on MacOS
```
no matching manifest for linux/arm64/v8 in the manifest list entries
```

---

It's going to be fixed with https://github.com/zeta-chain/ton-docker/pull/2